WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -126,6 +180,25 @@ private extension DashboardViewController {
 extension DashboardViewController: DashboardUIScrollDelegate {
     func dashboardUIScrollViewDidScroll(_ scrollView: UIScrollView) {
         hiddenScrollView.updateFromScrollViewDidScrollEventForLargeTitleWorkaround(scrollView)
+        showOrHideSubtitle(offset: scrollView.contentOffset.y)
+    }
+
+    private func showOrHideSubtitle(offset: CGFloat) {
+        if ServiceLocator.stores.sessionManager.defaultSite?.name != nil {
+            if offset > stackView.frame.height {
+                storeNameLabel.isHidden = true
+            }
+            if offset < -stackView.frame.height {
+                UIView.transition(with: storeNameLabel, duration: 0.2,
```

Comment:
It would be nice to have this transition on a constant enum, see `LoadingView.Layout` to allow us to easily find it and modify it in the future.

Reply:
Yep. Will be better if someone has to modify it in the future